docs/parallelism.md

```diff
@@ -0,0 +1,59 @@
+# Paralellism
+
+`--parallelism` is a flag available on `rolling-update`.  This flag is better thought of as "chunking" a deployment, instead
```

Comment:
don't need the comma at the end.

Reply:
I prefer the word partition instead of chunk :) https://en.wikipedia.org/wiki/Partition_of_a_set

Reply:
👍